sdk/core/core-rest-pipeline/test/throttlingRetryPolicy.spec.ts

```diff
@@ -21,13 +21,68 @@ describe("throttlingRetryPolicy", function () {
     sinon.restore();
   });
 
-  it("It should retry after a given number of seconds on a response with status code 429", async () => {
+  const defaultDurations = [0, 10 * 1000]; // milliseconds
+
+  defaultDurations.forEach((defaultDuration) => {
+    const headersWithDefaultDuration = [
+      {
+        "Retry-After": String(defaultDuration / 1000),
+      },
+      {
+        "retry-after-ms": String(defaultDuration),
+      },
+      {
+        "x-ms-retry-after-ms": String(defaultDuration),
+      },
+    ] as const;
+    headersWithDefaultDuration.forEach((headers) => {
```

Comment:
nit: did you consider using test-utils's `matrix`?

Reply:
Haven't considered